### PR TITLE
Disable captcha on the development pod by default

### PR DESCRIPTION
**Pull request by cmrd-senya**

### CODE REVIEW

**Comment by joebew42** [comment]
Hi, I think that you forgot to submit the pull request to the dependency module [puppet-diaspora](https://github.com/joebew42/puppet-diaspora). Thank you for your contribution!

**Comment by cmrd-senya** [comment]
But it's here, look!

File: puppet/modules/diaspora [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 4d505908f2ba9974783dfbf2e099a69b0212fd36
+Subproject commit 446d7476e51f7d7ed51b6926cd55e721f1c99c15
```
Comment (cmrd-senya) [inline puppet/modules/diaspora]: Here!

Comment (cmrd-senya) [inline puppet/modules/diaspora]: https://github.com/joebew42/puppet-diaspora/commit/446d7476e51f7d7ed51b6926cd55e721f1c99c15

Comment (joebew42) [inline puppet/modules/diaspora]: Ops, sorry :smile:

**Pull request merged**